ios-base/Networking/TargetType+Base.swift

```diff
@@ -49,4 +49,18 @@ extension TargetType {
   public func requestParameters(parameters: [String: Any]) -> Task {
     return .requestParameters(parameters: parameters, encoding: JSONEncoding.default)
   }
+
+  public func multipartData(from parameters: [String: Any]) -> [MultipartFormData] {
+    return parameters.map { (key: String, value: Any) -> MultipartFormData? in
+      let value = value
+      let key = "user[\(key)]"
```

Comment:
I think this should be more generic since all services will not be receiving the params under 'user'.

Reply:
good catch! forgot to parameterize that when moving the function to a helper